akka-operator/templates/040-deployment.yaml

```diff
@@ -71,6 +156,28 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+        {{- if .Values.provider.name }}
+        {{- if eq .Values.provider.name "gcp" }}
+        - name: ubbagent
+          image: "gcr.io/lightbend-public/akka-cloud-platform/ubbagent:{{ .Values.imageUbbagent.tag | default .Chart.AppVersion }}"
```

Comment:
The registry should be configurable:
https://github.com/lightbend/akka-cloud-platform-gcp/blob/main/apptest/deployer/manifest/tester.yaml#L40